package.json

```diff
@@ -263,6 +264,12 @@
     "@mapbox/mapbox-gl-draw": "1.3.0",
     "@mapbox/mapbox-gl-rtl-text": "0.2.3",
     "@mapbox/vector-tile": "1.3.1",
+    "@opentelemetry/api-metrics": "0.29.2",
+    "@opentelemetry/exporter-metrics-otlp-grpc": "^0.29.2",
```

Comment:
We should be able to set these all to 0.30.0 I think.

Reply:
done!